### ISSUE TITLE
Bab parameter on amlSearchByNumber

### ISSUE THREAD

**Issue by erscoder** [idanalyzer/id-analyzer-nodejs#3]
```
searchByIDNumber(documentNumber = "", country = "", dob = "") {
        if (documentNumber.length < 5)
        {
            throw new Error("Document number should contain at least 5 characters.");
        }
        return this._callAPI({
            "documentnumber": documentnumber,  // <- error
            "country": country,
            "dob": dob
        });
    }
```

should be

```
searchByIDNumber(documentNumber = "", country = "", dob = ""){
        if (documentNumber.length < 5)
        {
            throw new Error("Document number should contain at least 5 characters.");
        }
        return this._callAPI({
            "documentnumber": documentNumber,
            "country": country,
            "dob": dob
        });
    }
```

**Comment by erscoder** [comment]
@jeremyevith please can you fix this issue? We will need to use this feature

**Comment by jeremyevith** [comment]
This issue has been fixed in 1.2.7.
